src/app/(pages)/experience/page.tsx

```
@@ -0,0 +1,331 @@
+import { Badge } from '@/components/ui/badge'
+import { Button } from '@/components/ui/button'
+import Link from 'next/link'
+import {
+  type Experience,
+  experienceData,
+  impactMetrics,
+  experienceSummary
+} from '@/lib/data/experience'
+import { ResumeDownloadButton } from '@/components/pdf/ResumeDownloadButton'
+
+const Experience = () => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Experience'. Consider to delete it or rename it.

'Experience' is defined here:


(lint/suspicious/noRedeclare)